Documentation/Functions/Select.rst

```diff
@@ -154,7 +154,18 @@ pidInList
       Allows to disable the :sql:`pid` constraint completely. Requirements: 
       :ts:`uidInList` *must* be set or the table *must* have the prefix 
       "static\_\*".
+      
+   .. note::
```

Comment:
did you find a better solution? Should we point people there instead?

Reply:
No, I just think it should be stressed to check the page to be of the correct type.